src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -207,18 +207,10 @@ private String importDemoData(AdminHomePageData pageData)
         List<StudentAttributes> students = logic.getStudentsForCourse(courseId);
         List<InstructorAttributes> instructors = logic.getInstructorsForCourse(courseId);
 
-        for (CommentAttributes comment : comments) {
-            logic.putDocument(comment);
-        }
-        for (FeedbackResponseCommentAttributes comment : frComments) {
-            logic.putDocument(comment);
-        }
-        for (StudentAttributes student : students) {
-            logic.putDocument(student);
-        }
-        for (InstructorAttributes instructor : instructors) {
-            logic.putDocument(instructor);
-        }
+        logic.batchPutCommentDocuments(comments);
+        logic.batchPutFeedbackResponseCommentDocuments(frComments);
+        logic.batchPutStudentDocuments(students);
+        logic.batchPutInstructorDocuments(instructors);
```

Comment:
Once the methods are renamed, this will look much cleaner (all `logic.putDocuments`).